pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,224 @@
+package mysql
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+	"go.expect.digital/translate/pkg/tracer"
+)
+
+var mysqlRepo *Repo
+
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	viper.SetEnvPrefix("translate_mysql")
```

Comment:
This is a global viper setting. It won't work when we have `viper.SetEnvPrefix("translate")` in the CLI.

Reply:
Despite the name `TestMain()`, this function does not call `main.go main()` (The name must be `TestMain` to pass argument `*testing.M`). I know it is counter-intuitive, but that is one way to set up an integration test for repo: In this case, setting up the tracer and MySQL repo.

So shortly, for service integration tests, there is a separate viper instance, which does not have anything in common with the CLI viper instance, because they will never be called in the same runtime

Reply:
We should take into account it is the same runtime.

Reply:
Could you elaborate a little more about that ?
What would be the best course of action?